src/ui/CategorySelector/CategorySelector.svelte

```diff
@@ -1,7 +1,5 @@
 <script>
-  import DisplaySelectedCatFigures from "./DisplaySelectedCatFigures.svelte";
```

Comment:
I might be wrong but it looks like we could delete this file altogether?

Reply:
Only the import, the component is still needed, it is part of the V4 Design.
![Screenshot 2022-03-08 at 17 22 25](https://user-images.githubusercontent.com/70764326/157291610-92891fd2-de6a-4df0-ad27-eef4e5d787a9.png)